### PR TITLE
createrepo proposal

### PR DESCRIPTION
**Pull request by keichwa**
# Description
https://github.com/SUSE/spacewalk/issues/16827
https://bugzilla.suse.com/show_bug.cgi?id=1195294

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Review by jcayouette — APPROVED** [review]
Looks good thanks Karl!

File: modules/administration/pages/custom-channels.adoc [hunk 1]
```diff
@@ -78,12 +78,18 @@ For more information, see xref:administration:tshoot-registertradrhel.adoc[].
 ====
 
 
+Before you start with creating a software repository with the {webui} ensure that the [path]``repodata`` directory with the [path]``repomod.xml`` file exists in your repository directory.
+If [path]``repodata`` is missing, use the [command]``createrepo`` command:
+----
+createrepo /path/to/repo/directory
+----
+For example, replace [path]``/path/to/repo/directory`` with the [path]``/opt/mytools`` directory that is used in the following procedure.
```
[the remark this answers is not in the document]
Comment (mcalmer) [reply modules/administration/pages/custom-channels.adoc:86]: There are man pages for createrepo_c and reprepro

[the remark this answers is not in the document]
Comment (keichwa) [reply modules/administration/pages/custom-channels.adoc]: ok, makes sense.  I tried a more general description according to your proposal.  I hope it is better now.

File: modules/administration/pages/custom-channels.adoc [hunk 2]
```diff
@@ -78,12 +78,16 @@ For more information, see xref:administration:tshoot-registertradrhel.adoc[].
 ====
 
 
+Before you start with creating a software repository with the {webui} ensure that needed extra metadata is available.
+Tools such as [command]``createrepo`` and [command]``reprepro`` are useful in this regard.
+[command]``mgr-push`` can help with pushing a single RPM into a channel.
```
Comment (mcalmer) [inline modules/administration/pages/custom-channels.adoc:83]:
```suggestion
[command]``mgr-push`` can help with pushing a single RPM into a channel without creating a repository.
```

File: modules/administration/pages/custom-channels.adoc [hunk 3]
```diff
@@ -78,12 +78,16 @@ For more information, see xref:administration:tshoot-registertradrhel.adoc[].
 ====
 
 
+Before you start with creating a software repository with the {webui} ensure that needed extra metadata is available.
```
Comment (mcalmer) [inline modules/administration/pages/custom-channels.adoc:81]:
```suggestion
A pre requirement is, that you have a valid repository. Please check before you start adding a software repository to {productname} with the {webui},  that needed metadata are available. 
```

A repository for me is, what we have on disk with packages + metadata.
The repository in SUSE Manager is only including or using this repo. We do not really create it.
The text above is still not perfect. I am open for more suggestions.

**Review by mcalmer — CHANGES_REQUESTED** [review]
The direction is good, but we can make it even better.

File: modules/administration/pages/custom-channels.adoc (hunk 3 above)
Comment (keichwa) [reply modules/administration/pages/custom-channels.adoc]: I hope I got it right now.  Of course, I'm not sure whether my version actually is better...

**Comment by keichwa** [comment]
thank you, Michael!  @jcayouette or @0rnela one of you, please add your tic!

**Pull request merged**